package.json

```diff
@@ -20,8 +20,7 @@
   },
   "devDependencies": {
     "eslint": "^4.9.0",
-    "eslint-config-airbnb-base": "^12.1.0",
-    "eslint-plugin-import": "^2.8.0",
```

Comment:
why was import plugin removed?

Reply:
`eslint-plugin-import` was installed because it's a `peerDependency` of `eslint-config-airbnb-base`. We don't have any rules from `eslint-plugin-import` enabled separately on this repo.

Reply:
my response for this along the same lines as the above comment.

Reply:
I'm not sure what you mean -- it seems like since we're removing `eslint-config-airbnb-base`, it wouldn't make sense to keep `eslint-plugin-import` enabled because we wouldn't be using it at all.

Reply:
i mean we can use this individually by enabling rules from it. To start with just enable what is enabled inside airbnb.

Reply:
Oh, I see. That sounds fine to me.